tests/components/samsungtv/test_media_player.py

```diff
@@ -262,59 +282,66 @@ def mock_now():
     )
     state = hass.states.get(ENTITY_ID)
     # key and update called
-    assert remote.control.call_count == 2
-    assert remote.control.call_args_list == [call("KEY_VOLUP"), call("KEY")]
+    assert remote.control.call_count == 1
+    assert remote.control.call_args_list == [call("KEY_VOLUP")]
+    assert remote.close.call_count == 1
+    assert remote.close.call_args_list == [call()]
     assert state.state == STATE_ON
 
 
-async def test_send_key_autodetect_websocket(hass, remote):
-    """Test for send key with autodetection of protocol."""
+async def test_update_autodetect_websocket(hass, remote, mock_now):
+    """Testing update with autodetection of protocol."""
```

Comment:
Yesterday I moved the autodetect part in my PR to the config_flow. So the loser of our merge race has to fight with merge conflicts. :)
As I'm not able to do  the merge is not my part to choose the winner.

Reply:
I started this PR because of the mentioned issue and because there were no conflicts with your work (that was not dealing with `update` and `send_key`).
You're right... with `config_flow` the autodetect part will have no sense during normal activity.
IMHO Those tests will have to be rewritten in any case.
@MartinHjelmare, since you're reviewing @escoand's config flow work... what do you think?
I can surely wait and do the merge after the config flow part will be merged.

Reply:
Neither PR is ready for merging yet. Usually we merge the first PR that is ready.